docs/gallery/interactive/2024_workshop_daga.ipynb

```diff
@@ -0,0 +1,638 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "applied_filter  = rir * pf.dsp.pad_zeros(equalization, rir.n_samples-equalization.n_samples)\n",
+    "\n",
+    "# Compare original and equalized RTF\n",
+    "plt.figure()\n",
+    "pf.plot.freq(rir, color='grey', label='Original RTF')\n",
+    "ax = pf.plot.freq(applied_filter, label='Equalized RTF')\n",
+    "ax.legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Arithmetic operations\n",
+    "Analogously to `*`, the `/` operator implements a frequency domain division, the `@` operator implements a frequency domain matrix multiplication, and the `**` operator is the frequency domain power operator. Similary, `+` and `-` operators are available, which are domain agnostic."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filter Classes\n",
+    "\n",
+    "Pyfar implements **filters** as encapsulated class objects, similar to the Signal class.\n",
+    "Filter objects support storing sets of coefficients for FIR, IIR, as well as second order section IIR (SOS) filters.\n",
+    "They further facilitate piece-wise processing of audio signals while storing the filter state.\n",
+    "\n",
+    "A filter class object storing the previously created equalization filter only requires initializing the respective `pyfar.FilterFIR` class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "eq_filter = pf.FilterFIR(equalization.time, equalization.sampling_rate)\n",
+    "eq_filter"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example files\n",
+    "The [pyfar.signals.files](https://pyfar.readthedocs.io/en/stable/modules/pyfar.signals.files.html) module provides signals such as speech, castanets, drums, and guitar snippets, e.g., to have more natural signals for a quick demo of an audio system. Here, we choose a short anechoic guitar sample to simulate playback at the measured listener position by convolving it with the RIR. Because the sampling rates of the guitar sample and RIR do not match, the guitar signal is **resampled** first. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "guitar = pf.signals.files.guitar()\n",
+    "guitar = pf.dsp.resample(guitar, rir.sampling_rate)\n",
+    "reverberant_guitar = pf.dsp.convolve(guitar, rir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The equalization filter is applied to the guitar sample using the filter objects `process` method."
+   ]
```

Comment:
Maybe mention the process method in section "filter classes"?

Reply:
Good point, I moved the Example files section above Filter class, its part of this section now.